index.js

```diff
@@ -38,73 +38,116 @@ function f(d) {
     height: mobile ? undefined : VIDEO_HEIGHT,
-    timeout: 500,
-    renderPointCloud: true,
+    // renderPointCloud: !mobile, // can be used for mobile screens
+    renderPointCloud: false,
+    timeout: 300,
     renderCanvas: true,
-    renderFaceMesh: true,
+    // renderFaceMesh: true,
     onDetected: () => { touchCounter++; },
-    onRender: () => { stats.begin(); },
+    onRender: () => { 
+      stats.begin();
+      // clearInterval(loadingAnimation);
+      if (loading){
+        // let el = document.getElementById('loading-header');
+        // el.parentNode.removeChild(el);
+        $("#timesTouchedText").show();
+        $("#totalCount").show();
+        $("#title").show();
+        $("#footer").show();
+        let el = document.getElementById('loading-animation');
+        el.parentNode.removeChild(el);
+        // Copied here to ensure it only shows on load
+        const gui = new dat.GUI();
+        gui.add(state, 'frame timeout', 100, 1000).onChange((value) => { detector.update({ timeout: value }); });
+        loading = false;
+      }
+    },
     onRendered: (result) => {
       const d = result.minDistance;
       const handBox = result.handBox;
       const faceBox = result.faceBox;
       const deltaVolume = result.deltaVolume;
 
-      document.querySelector('#distance').innerText = d
-        ? `Closest ||p||: ${f(d.d)}, Δx: ${f(d.x)}, Δy: ${f(d.y)}, Δz: ${f(d.z)}`
-        : `Closest ||p||: Undefined`;
+      // document.querySelector('#distance').innerText = d
+      //   ? `Closest ||p||: ${f(d.d)}, Δx: ${f(d.x)}, Δy: ${f(d.y)}, Δz: ${f(d.z)}`
+      //   : `Closest ||p||: Undefined`;
 
-      document.querySelector('#intersection').innerText =
-        `Volume intersected: ${deltaVolume}`;
-      document.querySelector('#deltaCenter').innerText = d
-        ? `Center bounding box ||p||: ${f(Math.sqrt(Math.pow(handBox.xCenter - faceBox.xCenter, 2) + Math.pow(handBox.yCenter - faceBox.yCenter, 2) + Math.pow(handBox.zCenter - faceBox.zCenter, 2)))} Δx:${f(handBox.xCenter - faceBox.xCenter)} Δy:${f(handBox.yCenter - faceBox.yCenter)} Δz:${f(handBox.zCenter - faceBox.zCenter)}`
-        : `Center bounding box: Undefined`;
+      // document.querySelector('#intersection').innerText =
+      //   `Volume intersected: ${deltaVolume}`;
+      // document.querySelector('#deltaCenter').innerText = d
+      //   ? `Center bounding box ||p||: ${f(Math.sqrt(Math.pow(handBox.xCenter - faceBox.xCenter, 2) + Math.pow(handBox.yCenter - faceBox.yCenter, 2) + Math.pow(handBox.zCenter - faceBox.zCenter, 2)))} Δx:${f(handBox.xCenter - faceBox.xCenter)} Δy:${f(handBox.yCenter - faceBox.yCenter)} Δz:${f(handBox.zCenter - faceBox.zCenter)}`
+      //   : `Center bounding box: Undefined`;
 
-      document.querySelector('#detection').innerText =
-        `Detection: ${result.detected ? 'Yes' : 'No'}`;
+      // document.querySelector('#detection').innerText =
+      //   `Detection: ${result.detected ? 'Yes' : 'No'}`;
```

Comment:
Don't leave comment code. If we need to look back we can always look back with git. Also, I think we should leave another development html for the development of the core detector. But I am currently planning to leave that in another branch for now.